spec/support/contracts/json_schema_test/app/publish.mson

```diff
@@ -25,7 +25,7 @@ A post that only has a title, but no text body. Weird, right?
 - body: (string) - Body of the post
 - author_id: 2 (number, required) - User id of author
 - primary_tag: (App::Tag, optional) - Property of custom type
-- tags: (array[Tag]) - An array with a custom type
+- tags: (array[App::Tag]) - An array with a custom type
```

Comment:
Hmmm.... the way Lacerda has used this feature is:

```markdown
# LocalDefinition
- bar: (number) - This local definition defines an object that is not published by App, i.e. other services can not consume it directly

# App::PublishedDefinition
- id: (number) - This object is prefixed with App:: and other services can consume it
- foo: (LocalDefinition)
```

So I think it's actually useful. What's the reason for this change?